README.vi-VN.md

```diff
@@ -0,0 +1,16 @@
+<img width="150" height="150" align="left" style="float: left; margin: 0 10px 0 0;" alt="MS-DOS logo" src="https://github.com/Microsoft/MS-DOS/blob/master/msdos-logo.png">
+
+# Mã nguồn MS-DOS v1.25 và v2.0
+Repo này chứa mã nguồn và chương trình đã biên dịch của MS-DOS v1.25 và MS-DOS v2.0.
+
+Đây là những tệp tương tự được chia sẻ lần đầu tại [Bảo tàng Lịch sử Máy tính vào ngày 25 tháng 3 năm 2014]( http://www.computerhistory.org/atchm/microsoft-ms-dos-early-source-code/) và được tái xuất bản trong repo này để giúp mọi người dễ dàng tìm thấy, tham khảo bằng văn bản và công trình bên ngoài và cho phép khám phá và thử nghiệm những người quan tâm đến các hệ điều hành PC đầu tiên.
```

Comment:
@NamPNQ : Sorry I miss this in my last review. This sentence is quite unclear it could be "... dễ dàng tìm thấy, trích dẫn cho các bài viết và công trình ngoài và cho phép những người quan tâm đến các hệ điều hành PC đầu tiên khám phá và thử nghiệm." What do you think?

Reply:
:+1:

Reply:
I have a small change in your sentence: “...., được tái xuất bản trong repo này để giúp mọi người dễ dàng tìm thấy nó, trích dẫn nó cho các bài viết và công trình ngoài, cho phép những người quan tâm đến các hệ điều hành PC đầu tiên khám phá và thử nghiệm.”

Reply:
"...cho phép những người yêu thích hệ điều hành PC đầu tiên cơ hội khám phá và thử nghiệm." Does it look better?

Reply:
I agree. We will go with it. Thanks. 👍